cogs/etc.py

```diff
@@ -395,5 +402,12 @@ def to_string(c):
             return await ctx.send('Output too long to display.')
         await ctx.send(msg)
 
-def setup(bot):
-    bot.add_cog(Etc())
+    @commands.command(name="Profilepic", aliases=['pfp'])
```

Comment:
I really dont see the reason for this to be added

Reply:
why dont they have a /shrug on github lmao. simply